src/main/frontend/util/property.cljs

```diff
@@ -284,7 +284,7 @@
         content (reduce (fn [content key]
                           (remove-property format key content)) content built-in-properties*)]
     (if (= format :org)
-      (string/replace-first content ":PROPERTIES:\n:END:" "")
+      (string/replace-first content ":PROPERTIES:\n:END:\n" "")
```

Comment:
We can replace ":PROPERTIES:\n:END:\n" with `(re-pattern ":PROPERTIES:\n:END:\n*")` so that it works for these two cases:
1. the content doesn't have `\n` after `:PROPERTIES:\n:END:`
2. the content has multiple `\n`s after `:PROPERTIES:\n:END:`

Reply:
Great, this one will cover more cases. Do I need to submit the newer one again? Would you please add the *?

Reply:
@llcc Yes, let me add that. Merging now.